plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -5,6 +5,14 @@ class Pod
       INIT_CONTAINER_KEY = :'pod.beta.kubernetes.io/init-containers'
       INGORED_AUTOSCALE_EVENT_REASONS = %w[FailedGetMetrics FailedRescale].freeze
 
+      def self.init_containers(pod)
+        containers = pod.dig(:spec, :initContainers) || []
+        if json = pod.dig(:metadata, :annotations, Kubernetes::Api::Pod::INIT_CONTAINER_KEY)
+          containers += JSON.parse(json, symbolize_names: true)
```

Comment:
won't this end up with duplicates if both are specified (e.g. on 1.7)?

Reply:
yeah kinda ... migrated pods will have both, maybe some other conditions ... I'll add a uniq

Reply:
I have a feeling kubernetes actually picks one or the other rather than merging, but i'm not sure which / can't find documentation on that.

Reply:
would depend on the cluster I guess ... but does not hurt to try to find logs for both I'd guess